site/app/views/AutoGradingView.php

```diff
@@ -608,17 +609,14 @@ public function showPeerResults(TaGradedGradeable $ta_graded_gradeable, bool $re
         $annotation_path = FileUtils::joinPaths($this->core->getConfig()->getCoursePath(), 'annotations', $gradeable->getId(), $id, $active_version);
         $annotated_file_names = [];
         if (is_dir($annotation_path) && count(scandir($annotation_path)) > 2) {
-            $first_file = scandir($annotation_path)[2];
-            $annotation_path = FileUtils::joinPaths($annotation_path, $first_file);
-            if (is_file($annotation_path)) {
+            //$first_file = scandir($annotation_path)[2];
+            //$annotation_path = FileUtils::joinPaths($annotation_path, $first_file);
+            if (is_dir($annotation_path)) {
                 $dir_iter = new \DirectoryIterator(dirname($annotation_path . '/'));
```

Comment:
Use a [`FilesystemIterator`](https://www.php.net/manual/en/class.filesystemiterator.php) which will skip dots for you.